api/inc/vmpu_exports.h

```diff
@@ -147,14 +147,42 @@ typedef struct {
 typedef struct {
     uint32_t magic;
     uint32_t version;
+
+    /* contains all volatile memory of the box */
+    uint32_t bss_size;
```

Comment:
* Is there a relationship between `bss_size` and the other elements below? From the comments it looks like it's `stack_size + context_size + heap_size + <something>`.
* Which of these sizes is expected/advised to be already rounded up?